app/services/database_migration_service.py

```
@@ -0,0 +1,687 @@
+                {
+                    "$set": {
+                        "nfl_context": {},
+                        "sentiment_weights": {},
+                        "emotion_scores": {},
+                        "aspect_sentiments": {},
+                        "keyword_contributions": {},
+                    }
+                },
+            )
+
+            return True
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)